Automattic/jetpack
# VideoPress: Edit attachment styles to align share and ratings.

Fixes #5627

#### Changes proposed in this Pull Request:

- Add in styles for share and ratings to align checkboxes and radio buttons.
- Move style enqueue and register into the admin_head action

#### Testing instructions:

- Verify that when view a VideoPress attachment from the media library that the buttons are aligned.

![](https://cldup.com/9Ex3-8lzuq.thumb.png)
